pkg/lib/packagekit.js

```diff
@@ -256,12 +256,12 @@ export function cancellableTransaction(method, arglist, progress_cb, signalHandl
                 if ("Status" in props)
                     status = props.Status;
                 progress_data.waiting = allow_wait_status && (status === Enum.STATUS_WAIT || status === Enum.STATUS_WAITING_FOR_LOCK);
-                if ("Percentage" in props && props.Percentage <= 100)
-                    progress_data.percentage = props.Percentage;
                 if ("AllowCancel" in props)
                     progress_data.cancel = props.AllowCancel ? cancel : null;
```

Comment:
But it will not now notify on changes like `progress_data.cancel` and such, won't it?

Reply:
Correct, but  really enabling/disabling the cancel operation 1 second before or after does not cause a big problem right?

Let me think about this a bit. Maybe there is a better solution.

Reply:
Ok I found  a cleaner fix.